core/src/main/java/org/apache/iceberg/rest/RESTSessionCatalog.java

```diff
@@ -943,11 +968,11 @@ private static Cache<String, AuthSession> newSessionCache(Map<String, String> pr
         .build();
   }
 
-  private Cache<TableOperations, FileIO> newFileIOCloser() {
+  private Cache<Object, FileIO> newFileIOCloser() {
```

Comment:
Since this doesn't need to create `FileIO` instances for views, I don't think this change is needed. Right?

Reply:
agreed, we can revert these changes and we don't need to track FileIO here for views